README.md

```diff
@@ -17,84 +17,117 @@ Each day has a folder containing a README.md file with instructions for the day,
 
-### Setting Up Terminal Access for Atom
+## Setting Up Terminal Access for Atom
 
 Install the shell commands for Atom. Open Atom, drop down the `Atom` menu in the top left corner of your screen, and click on `Install Shell Commands`. Atom should now be enabled from your command line.
 
-To confirm that atom is working from your command line, open a terminal with Spotlight search (`Command + Space`), type "terminal" and hit enter. Type `atom .` in your terminal. If it is setup correctly, the atom editor will automatically open. If it does not open atom and an error occurs instead, try entering this in the command line: `ln -s /Applications/Atom.app/Contents/Resources/app/atom.sh /usr/local/bin/atom` and try the first command (`atom .`) again.
+![Install Shell Commands](/images/install_shell_commands.jpg)
+
+To confirm that atom is working from your command line, open a terminal with Spotlight search (`Command + Space`), type `terminal` and hit enter. Type `atom .` in your terminal. If it is setup correctly, the atom editor will automatically open. If it does not open atom and an error occurs instead, try entering this in the command line: `ln -s /Applications/Atom.app/Contents/Resources/app/atom.sh /usr/local/bin/atom` and try the first command (`atom .`) again.
 
 Atom also offers a number of different options and packages that you can customize to your liking. [This](https://www.youtube.com/watch?v=WWwBQQOGllo&list=PLYzJdSdNWNqwNWlxz7bvu-lOYR0CFWQ4I) series of videos will walk you through many of them if you'd like to dive deeper.
 
-### GitHub SSH Keys
+## GitHub SSH Keys
 
 SSH keys are a more secure and convenient way of authenticating than typing in our password every time we want to interact with Github.
 
--   Generate a new key by running (You should use the email associated with your GitHub account)
+#### 1. Generate a new key by running:
+
+(You should use the email associated with your GitHub account)
 
-```bash
+```
 $ ssh-keygen -t rsa -C "johndoe@example.com"
 ```
 
-When you're prompted to "Enter a file in which to save the key," press Enter. This accepts the default file location.
-When asked to enter a password, hit enter.
+When you're prompted to "Enter a file in which to save the key," press Enter. This accepts the default file location, which is `~/ssh/id_rsa`
+
+When asked to enter a password, hit enter, which means "no password".
+
+You'll see a confirmation message that looks like:
+
+```
+Your identification has been saved in id_rsa.
+Your public key has been saved in id_rsa.pub.
+The key fingerprint is:
+SHA256:C3AB8FF13720E8AD9047DD39466B3C8974E592C your@email_address.com
+The key's randomart image is:
++---[RSA 2048]----+
+| .       =   ..  |
+|o . . o + = ..   |
+| =.o o o o o  .  |
+|+ +o. .  ..  . . |
+|.+E  .  S   o o..|
+|..     .  .o . .+|
+|        o  oo .o+|
+|       . o  ==o.=|
+|        . .+=B=o |
++----[SHA256]-----+
```

Comment:
For the security-conscious, in these code snippets, I generated fake keys and never actually added them to my github profile.